apps/server/lib/lexical/server/provider/handlers/hover.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Lexical.Server.Provider.Handlers.Hover do
+  alias Lexical.Protocol.Requests
+  alias Lexical.Protocol.Responses
+  alias Lexical.Protocol.Types.Hover
+  alias Lexical.Protocol.Types.Markup
+  alias Lexical.RemoteControl
+  alias Lexical.Server.CodeIntelligence.Entity
+  alias Lexical.Server.Provider.Env
+
+  require Logger
+
+  def handle(%Requests.Hover{} = request, %Env{} = env) do
+    maybe_hover =
+      with {:ok, {:module, module}} <- Entity.resolve(request.document, request.position),
+           {:ok, doc_content} <- module_doc_content(env.project, module) do
+        module_name = module |> to_string() |> String.replace_prefix("Elixir.", "")
```

Comment:
I find we do this a lot in our code. Perhaps we should make a utility for splitting modules (both elixir and erlang) and turning them into strings.

Reply:
Any thoughts on where to put this and what to name the utility module?

Reply:
I'd put it in common, I'm flexible on the name, `Lexical.Ast.Module` maybe?